applications/roloi.md

```diff
@@ -0,0 +1,161 @@
+
+He’s the growth lead of [DeFi Argentina](https://twitter.com/DeFiArgentina) a non-profit project that helps children’s food banks in Argentina by collecting donations in cryptocurrencies. 
+
+He’s also an **OAK ambassador** since Nov 2022.
+
+Hernán is part of the **Roloi team** as CGO to help with the growth, research and networking strategy, leveraging his knowledge about DeFi and communities.
+
+### Team Code Repos
+* [NeoPower Github Repository](https://github.com/NeoPower-Digital)
+
+## Development Status 
+
+We are working with the library [inkathon](https://github.com/scio-labs/inkathon) to simplify UI development and connection to the blockchain, this allows developers to focus more on solving actual problems.
+
+On the other hand we’ve been exploring OAK **code examples** and we identified some improvements that need to be built in the form of project templates to push forward the creation of new products using OAK.
+
+We had the opportunity to pitch Roloi to the OAK team and our plans to **build the core infrastructure needed to configure recurring transactions from our ink! smart contract.**
+
+## Development Roadmap 
+
+### Overview
+
+* **Total Estimated Duration:** 2 months
+* **Full-Time Equivalent (FTE):** 4
+* **Total Costs:** 50,000 USD
+
+### Milestone 1 — Automation UI Template
+
+* **Estimated duration:** 1 month
+* **FTE:** 4
+* **Costs:** 25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | The milestone will include technical documentation of the UI project. |
+| 0c. | Testing Guide | Core functions will be covered by unit tests to ensure functionality and robustness. We will describe how to run these tests in the docs. |
+| 1. | UI Source Code | We will deliver a Next.js UI project that enables the execution of a simple recurring task (on-chain event) and works as a template to be reused for any team that needs automated tasks. Tools and libraries: Next.js, Typescript, Material UI, XState. *The scope of this Milestone will cover only testnet chains.* |  
+| 1a. | Wallet connection | The UI will connect with Polkadot Wallets using inkathon functionalities. |  
+| 1b. | Chain connections | The UI will allow the user to connect with different parachains (Astar, Mangata, etc) using inkathon functionalities. The chains must be connected with OAK. |  
+| 1c. | Proxy Accounts | The UI will solve the configuration for the proxy accounts: If they don’t exist for the connected account, create them, transfer funds to each proxy account and assign them the appropriate permissions. |  
+| 1d. | Create recurring task | The UI will include a simple event configured to be easily triggered recurrently on the selected chain. |  
+| 1e. | Cancel task | The UI will include a form to cancel a specific OAK task ID. |  
```

Comment:
One crucial aspect that is currently missing is the display of tasks. Without a list of tasks, users will not be able to determine which ones to cancel.

I suggest reallocating some of the effort currently dedicated to unit test cases towards building a display page. This would enable users to view their tasks and make more informed decisions about which ones to cancel.

Reply:
Great suggestion, we can include that.